src/actinia_core/rest/resource_management.py

```diff
@@ -352,21 +353,22 @@ def _create_ResourceDataContainer_for_resumption(
             is None
         ):
             return None, None, None
-        location = re.findall(r"locations\/(.*?)\/", post_url)[0]
+        # TODO project project
```

Comment:
What does this TODO mean?

Reply:
Das war nur die Erinnerung für mich. Habe gerade die aktuelle Version hochgeladen